topics/introduction/slides/introduction.html

```diff
@@ -111,63 +124,99 @@
 
 Tools can be viewed as tiny LEGO pieces: each one solves a specific problem, and they can be combined together to build complex analysis pipelines.
 
+
+---
+
+### ToolShed
+
+.image-50[![](../images/galaxy_ts.png)]
+- Free "app" store: [Galaxy Tool Shed](https://toolshed.g2.bx.psu.edu/)
+  - Thousands of tools already available
+  - Most software can be made available
```

Comment:
```suggestion
  - Most software can be integrated
```
(available is used already in the next line)